etcd_helper/etcdfuncs.go

```diff
@@ -0,0 +1,113 @@
+package etcd_helper
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/coreos/etcd/client"
+)
+
+type Etcdreader struct {
+	etcdServiceURL string
+}
+type Etcdwriter struct {
+	etcdServiceURL string
+}
+
+func (o Etcdreader) get(resourceKey string) (string, error) {
+	cfg := client.Config{
+		Endpoints: []string{o.etcdServiceURL},
+		Transport: client.DefaultTransport,
+	}
+	c, err := client.New(cfg)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Instead of log.Fatal(err), lets return nil, err

Reply:
Actually, lets do what you are doing in line 32.

Reply:
Done